Analyses/src/CRYGenPlots_module.cc

```diff
@@ -115,7 +147,34 @@ void mu2e::CRYGenPlots::analyze(art::Event const & e)
 
     HepLorentzVector mom4 = p.momentum();
     Hep3Vector mom3(mom4.px(), mom4.py(), mom4.pz());
-    Hep3Vector yMom3(mom4.pz(), mom4.px(), mom4.py()); // for theta, phi
```

Comment:
Yes, this basically uses the "cosmic" coordinate system instead of the "Mu2e" coordinate system because it makes more sense. I think I do this only here, can you point me the the other places where I do this eventually?